nym-wallet/src/components/FeeWarning.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Warning } from '@mui/icons-material';
+import { FeeDetails } from '@nymproject/types';
+import { Alert, AlertTitle } from '@mui/material';
+
+export const FeeWarning = ({ fee, amount }: { fee: FeeDetails; amount: number }) => {
+  if (fee.amount && +fee.amount.amount > amount) {
```

Comment:
`+fee.amount.amount` typo?

Reply:
`fee.amount.amount` is of type string. The `+` will convert it to a number :)